Flowery/Source Files/Flows/Main/Products/List/ProductListViewController.swift

```diff
@@ -86,11 +86,22 @@ extension ProductsListViewController: UITableViewDataSource, UITableViewDelegate
         let cell = tableView.dequeueReusableCell(withIdentifier: ProductTableViewCell.reuseIdentifier, for: indexPath) as! ProductTableViewCell
         let product = products[indexPath.row]
         let imageURL = viewModel.imageURL(for: product)
+        let watering: String
+        switch product.watering {
+        case .average:
+            watering = "🚰🚰"
+        case .frequent:
+            watering = "🚰🚰🚰"
```

Comment:
i think this emoji could fit better 🌊

Reply:
that's nice idea; however, it could confuse people and they could use salty water for their plants 🧐

Reply:
Hahaha true! Can't disagree 

Let's leave the decision up to a PR owner 🤝